app/pages/viewuser/confirmation/contextCreator.js

```diff
@@ -0,0 +1,10 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../../config';
+
+export const getContext = ({
+  user, organisationId = 1, status,
+}) => ({
+  ...manifest[status],
+  title: `${user.name}${manifest[status].title}`,
```

Comment:
do you think it would be possible to use that `addParamsToManifest` and keep {{ user.name }} in there?

Reply:
ok